bigbluebutton-html5/package.json

```diff
@@ -32,10 +32,10 @@
     "react-tabs": "^0.8.2",
     "react-toggle": "^2.2.0",
     "redis": "^2.6.2",
-    "underscore": "~1.8.3",
     "winston": "^2.3.1",
     "xml2js": "^0.4.17",
-    "xmlhttprequest": "^1.8.0"
+    "xmlhttprequest": "^1.8.0",
```

Comment:
I noticed that this package is not used anywhere. It's imported in the breakoutJoinURL.js only, but never used.